src/securityPolicy.c

```diff
@@ -703,6 +708,8 @@ static inline security_policy_t _ordinaryWitnessPolicy(const bip44_path_t* path,
 {
 	switch (bip44_classifyPath(path)) {
 	case PATH_ORDINARY_SPENDING_KEY:
+		DENY_IF(violatesAndStoreSingleAccount(path));
+	// intentional fallthrough
```

Comment:
Let's not use any fallthrough here, security policies is not a place for anything error-prone.

Reply:
Is there any reason to *not* check for single account violation in case of PATH_ORDINARY_STAKING_KEY?

Reply:
IMHO staking paths should also be checked. They are being checked on Trezor.